_ko/tour/mixin-class-composition.md

```diff
@@ -10,7 +10,7 @@ num: 6
 language: ko
 
 next-page: higher-order-functions
-previous-page: traits
+previous-page: tuples
 ---
 
 _단일 상속_ 만을 지원하는 여러 언어와는 달리, 스칼라는 더욱 일반화된 시각에서 클래스를 재사용한다. 스칼라는 새로운 클래스를 정의할 때 _클래스의 새로운 멤버 정의_ (즉, 슈퍼클래스와 비교할 때 변경된 부분)를 재사용할 수 있다. 이를 _믹스인 클래스 컴포지션_ 이라고 부른다. 이터레이터를 추상화한 다음 예제를 살펴보자.
```

Comment:
This should be replaced by the stub to indicate it hasn't been translated yet.

Reply:
There is some translation there, not sure though if it corresponds to English version.

Reply:
My mistake,  I got confused by the fact that this page is actually about mixins, and it's just the "previous" pointer that's being updated.